### PR TITLE
Add content for ICH example

### PR DESCRIPTION
**Pull request by mritunjaysharma394**
Adds content for our example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,99 @@
-# Node.js Express Web Service using standard `node` image
+# Hardening Docker Workloads using Slim CLI
```
Comment (iximiuz) [inline README.md]: Primer: Node.js Application Hardening

File: README.md [hunk 2]
```diff
@@ -1,3 +1,99 @@
...
 
-TO-DO
+## The Problem and the Magic 🪄
```
Comment (iximiuz) [inline README.md]: I personally love it, but not every engineer out there appreciates the "magical" part of our solution. Let's stick with "The Problem and The Solution"

File: README.md [hunk 3]
```diff
@@ -1,3 +1,99 @@
...
+Hello folks and if you are reading this, you might have already flashed a gaze at the picture below but before we come to that, let's take a look at the repository and it basically consists of just a simple (but huge) node app and a Dockerfile majorly. Now let's come back to the image (pun intended ofcourse) below again. The left side is what how the original image of our node app looks like and as visible, its huge 995 MB. Now what if we say that you can get the same app's hardened image with size minified to just 89 MB and vulnerabilites reduced by 99.67% and that too in just few simple steps? Sounds magical right? Well, so says the image on your right and that's the magic we are going to talk about today!
```
Comment (iximiuz) [inline README.md]: General tech writing advice: cut the fluff 🙈 

I totally get it - it's your style. But unless you're writing an entertaining-only piece, it's usually a good idea to remove all the words, phrases, and even sentences that can be removed w/o reducing the amount of information you need to convey. It's a pity. But our writing competes with soo many distractions that it just must be concise.

File: README.md (hunk 1 above)
Comment (iximiuz) [inline README.md]: It's kinda handy to have a sentence per line in markdown files. Helps with reviews and change tracking.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,99 @@
...
 
-TO-DO
+## The Problem and the Magic 🪄
+Hello folks and if you are reading this, you might have already flashed a gaze at the picture below but before we come to that, let's take a look at the repository and it basically consists of just a simple (but huge) node app and a Dockerfile majorly. Now let's come back to the image (pun intended ofcourse) below again. The left side is what how the original image of our node app looks like and as visible, its huge 995 MB. Now what if we say that you can get the same app's hardened image with size minified to just 89 MB and vulnerabilites reduced by 99.67% and that too in just few simple steps? Sounds magical right? Well, so says the image on your right and that's the magic we are going to talk about today! 
+
+![image](https://user-images.githubusercontent.com/45476902/218093055-50a44810-db1a-43fd-a71d-909e521d4a55.png)
+
+
+### Prerequisites
+* A fresh version of the Slim CLI is installed and configured as described on this [portal](https://portal.slim.dev/cli)
```
Comment (iximiuz) [inline README.md]: `nstalled and configured` can be just a link on itself (and then trim the rest of the sentence).

File: README.md [hunk 5]
```diff
@@ -1,3 +1,99 @@
...
+
+* Connector configured with Slim SaaS that will help you get your target image
```
Comment (iximiuz) [inline README.md]: [A container registry connector configured](https://portal.slim.dev/connectors).

File: README.md (hunk 2 above)
Comment (iximiuz) [reply README.md]: Great to have this picture that gives the reader an almost instant understanding of the hardening benefits. However, we shouldn't probably be repeating the info from the picture in the text. Instead, I'd focus on the actual problem and the solution. Something along the lines of:

Creating high-quality container images is hard. How to choose an optimal base image? How to configure a multistage build to make the end slim? How to keep the vulnerability scanners calm? One must be a true container expert to get it right. However, there might be an alternative way.

Automated container image hardening:

- Build an image FROM your favorite base.
- Instrument the image with our sensors.
- Run an instrumented container "probing" its functionality.
- Build a hardened version of the image using collected intelligence.

(and here your screenshot goes)

File: README.md [hunk 6]
```diff
@@ -1,3 +1,99 @@
...
+
+* Docker (Engine or Desktop) is installed locally.
+
+### Demo
```
Comment (iximiuz) [inline README.md:28]: Probably a good idea to have the visualization of the demo here. You can reuse my drawing from here https://slim-ai.atlassian.net/wiki/spaces/ENG/pages/1434845199/How+to+Harden+Docker+Workloads+using+Thin+Slim+CLI. Or make your own.

**Review by iximiuz — CHANGES_REQUESTED** [review]

**Comment by mritunjaysharma394** [comment]
Thanks for the review @iximiuz, tried to improve most of it :smile:

File: README.md (hunk 3 above)
Comment (mritunjaysharma394) [reply README.md]: Really liked this advice :pleading_face:

File: README.md [hunk 7]
```diff
@@ -1,3 +1,111 @@
-# Node.js Express Web Service using standard `node` image
+# Primer: Node.js Application Hardening
 
-TO-DO
+## The Problem and the Solution 
+
+Creating high-quality container images is hard. How to choose an optimal base image? How to configure a multistage build to make the end slim? How to keep the vulnerability scanners calm? One must be a true container expert to get it right. However, there might be an alternative way.
```
Comment (iximiuz) [inline README.md]: `make the end slim` -> `make the end image slim`

P.S. One sentense - one line 😉

File: README.md [hunk 8]
```diff
@@ -1,3 +1,111 @@
...
+
+![image](https://user-images.githubusercontent.com/45476902/218093055-50a44810-db1a-43fd-a71d-909e521d4a55.png)
+
+
+### Prerequisites
+* A fresh version of the Slim CLI [installed and configured](https://portal.slim.dev/cli)
+
+* [A container registry connector configured](https://portal.slim.dev/connectors)
+
+* Docker (Engine or Desktop) is installed locally.
```
Comment (iximiuz) [inline README.md]: Let's replace it with `An ability to run the instrumented image (e.g., using Docker or Kubernetes)`. The word "local" may mislead people - the coolness of the ICH approach is that we don't really require running anything locally.

File: README.md [hunk 9]
```diff
@@ -1,3 +1,111 @@
...
+
+### Demo
+
+Before we move to the steps, let's visualise with this great image from [Ivan](https://github.com/iximiuz) on how the flow looks like!
```
Comment (iximiuz) [inline README.md]: No need to mention me here :))

File: README.md [hunk 10]
```diff
@@ -1,3 +1,111 @@
...
+```sh
+$ slim instrument \
+  --include-path /etc/passwd \
+  --stop-grace-period 999s  \
+   ghcr.io/mritunjaysharma394/node-app:latest
+```
+
+**NOTE**: Make sure the instrumented image, in our case, `ghcr.io/mritunjaysharma394/node-app:latest-slim-instrumented` is available through the connector.
+
+> Sometimes, the slim instrument command fails to find the target image even when it’s visible on the user portal. It is caused by the suboptimal heuristic that we use to map an image to a connector. You can mitigate it by explicitly specifying the connector ID using the --target-image-connector, --instrumented-image-connector, and --hardened-image-connector flags. The correspond connector ID can be copied from the connector editing page on the portal.
```
Comment (iximiuz) [inline README.md]: Not a good look for us 🙈  Let's remove this warning.

File: README.md [hunk 11]
```diff
@@ -1,3 +1,111 @@
...
+
+### Demo
+
+Before we move to the steps, let's visualise with this great image from [Ivan](https://github.com/iximiuz) on how the flow looks like!
+![image](https://user-images.githubusercontent.com/45476902/218159028-d2b21334-bfeb-45dd-8d2d-725fbe3d3520.png)
+
+
+#### Step 1: Instrument  🕵️
+
+The first step involves instrumenting the image and simply speaking it means that the Slim engine's sensor is going to act like an intelligence agent (the Container's James Bond) to collect data for the further probe  🕵️
```
Comment (iximiuz) [inline README.md]: You know James Bond is known for destroying everything he interacts with, right? :))

File: README.md (hunk 1 above)
Comment (mritunjaysharma394) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (mritunjaysharma394) [reply README.md]: Yes, thanks :))

**Comment by mritunjaysharma394** [comment]
Merging this too based on earlier discussion

**Pull request merged**